Gemfile.lock

```diff
@@ -44,35 +44,35 @@ GEM
     loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.2)
+    method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
+    minitest (5.14.0)
     money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.8)
+    rack (2.2.2)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    railties (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
     thor (1.0.1)
     thread_safe (0.3.6)
-    tzinfo (1.2.5)
+    tzinfo (1.2.6)
```

Comment:
### [tzinfo](https://tzinfo.github.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.5 → 1.2.6

[_~~change-log~~, [source-code](https://github.com/tzinfo/tzinfo)_]





<details>
<summary>Commits</summary>

A change of **27** commits. See the full changes on [the compare page](https://github.com/tzinfo/tzinfo/compare/v1.2.5...v1.2.6).

These are the first 10 commits:
- (cccd70b) [Restrictions on timezones only apply to older (pre-1.9) Ruby…](https://github.com/tzinfo/tzinfo/commit/cccd70b9448e7ba482b83f4dbddc6d2d60af4ee8)
- (fe9b204) [Return the correct seconds since the epoch value for strftim…](https://github.com/tzinfo/tzinfo/commit/fe9b2049eb44bb16841f32c998625e4b4dc0790e)
- (3b840ca) [Fix a documentation typo.](https://github.com/tzinfo/tzinfo/commit/3b840ca962f5755bfef94c148776bd26e1b89662)
- (6f07fa5) [Update to the latest Ruby, JRuby and Rubinius releases.](https://github.com/tzinfo/tzinfo/commit/6f07fa5f8709103270f989d5fdd8f8328a600c7a)
- (7a660fc) [Skip tests that fail due to Ruby bug 14060 on Ruby 2.4.4.](https://github.com/tzinfo/tzinfo/commit/7a660fcc1f198fd8b26aaa3a96d31accfeb57c65)
- (f61cfe6) [Test that RUBY_ENGINE is defined.](https://github.com/tzinfo/tzinfo/commit/f61cfe6d873eabcc06765ad7d0d8bdd3f33790a6)
- (769e268) [Fix SecurityErrors when loading data in safe mode.](https://github.com/tzinfo/tzinfo/commit/769e26800160ee50459606fea9eba0b885c45e4f)
- (9fd7882) [Remove the unused REQUIRE_PATH constant from RubyDataSource.](https://github.com/tzinfo/tzinfo/commit/9fd78820d83b7ba42a23665e58de57c6cd8e5619)
- (c1f3b2a) [Fix test failures on Ruby 1.8.7 caused by DateTime issues.](https://github.com/tzinfo/tzinfo/commit/c1f3b2ab1ad917c5d629b1222db1435f14a51e7a)
- (3d4c491) [Suppress deprecation warnings due to Object#untaint on Ruby …](https://github.com/tzinfo/tzinfo/commit/3d4c491f472301e0ae5e7c446b96a3d10591df2c)


</details>